src/components/ContactForm/ContactForm.tsx

```diff
@@ -73,9 +73,10 @@ export default function ContactForm({
             <TextInput
               label="Subject"
               placeholder="eg. Job offer - let&rsquo;s talk!"
-              value={formik.values.subject}
+              value={formik.values.subject || "Job offer - let's talk!"}
```

Comment:
**issue:** You don't set initial form value this way - it should be set in initial formik values